js_modules/dagster-ui/packages/ui-core/src/insights/InsightsIcon.tsx

```diff
@@ -15,7 +15,9 @@ interface InsightsIconProps {
 export const InsightsIcon = (props: InsightsIconProps) => {
   const {name, color = Colors.accentPrimary(), size = 16} = props;
   if (IconNames.includes(name as IconName)) {
-    return <Icon name={name as IconName} style={{marginLeft: 0}} color={color} />;
+    return (
+      <Icon name={name as IconName} style={{marginLeft: 0}} color={color} size={size as any} />
```

Comment:
I've needed to do this in a few other places, in general it would be nice for `size` to be `14 | 18 | number` (or something like that) so that we can use custom icon sizes where we want to more easily.

Reply:
Yeah, in this case we weren't passing the size down at all so we were just silently ignoring it 😓